testing/features/search.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Search components
+
+  The Search component allows users to search for the desired term and consists
+  of a search field and a button. The variant 'Search with value' has a
```

Comment:
lets omit this description here, as this isn't parsed in the html report proper. And amend the cukes below